### PR TITLE
updates for BZ1728350

### PR DESCRIPTION
**Pull request by lamek**

### CODE REVIEW

**Comment by lamek** [comment]
@timflannagan1 PTAL.

File: modules/metering-store-data-in-s3.adoc [hunk 1]
```diff
@@ -119,3 +119,5 @@ If you left `spec.storage.hive.s3.createBucket` set to `true`, or unset, then yo
     ]
 }
 ----
+
+Metering does not manage or delete any s3 bucket data. When uninstalling metering, any s3 buckets created for metering storage must be manually cleaned up.
```
Comment (timflannagan1) [inline modules/metering-store-data-in-s3.adoc]: I think we should move this higher up to avoid the situation where a user skips over some key information.

Comment (timflannagan1) [reply modules/metering-store-data-in-s3.adoc]: I think we should also specify that this applies to any s3 bucket used for metering and not just buckets created by Metering, i.e. "..., any s3 buckets used to store metering data must be manually cleaned up."

Comment (lamek) [reply modules/metering-store-data-in-s3.adoc]: Sounds good. I made some updates per your feedback.

File: modules/metering-store-data-in-s3.adoc [hunk 2]
```diff
@@ -5,6 +5,8 @@
 [id="metering-store-data-in-s3_{context}"]
 = Storing data in Amazon S3
 
+Metering can use an existing Amazon S3 bucket or create a new bucket for storage. Metering does not manage or delete any S3 bucket data. When uninstalling metering, any S3 buckets used to store metering data must be manually cleaned up.
```
Comment (kalexand-rh) [inline modules/metering-store-data-in-s3.adoc]: Can you uninstall metering? There's no topic for it, and this information would be far more helpful in an uninstalling topic.

~new~

Consider refocusing the last sentence on the fact that you must manage the S3 data yourself and using uninstalling as an example.

**Review by kalexand-rh — COMMENTED** [review]
I'd consider rewriting the module to better match the procedure template, and I think that documenting how to uninstall the metering service would help address the root cause.

File: modules/metering-store-data-in-s3.adoc (hunk 2 above)
Comment (lamek) [reply modules/metering-store-data-in-s3.adoc]: Great suggestion. We were discussing the same thing on the metering team. I'm going to do a separate PR for a section on "Deleting Reports". I'll add in a section for "Uninstalling Metering."

**Pull request merged**